projects/RabbitMQ.Client/client/api/IConnection.cs

```diff
@@ -143,9 +143,9 @@ public interface IConnection : INetworkConnection, IDisposable
         /// <see cref="IConnection"/>, then this event will be signalled whenever one
         /// of those event handlers throws an exception, as well.
         /// </remarks>
-        event EventHandler<CallbackExceptionEventArgs> CallbackException;
+        event AsyncEventHandler<CallbackExceptionEventArgs> CallbackException;
```

Comment:
Do you think the event names should end with `Async` to be consistent with the rest of the async naming?

Reply:
I'm not a huge fan of the suffix, so I let you decide ;)

Reply:
> Yeah, let's have the suffix everywhere, to be consistent. I can get that if you'd like, just let me know.

Please go ahead